addons/mass_mailing/i18n/mass_mailing.pot

```diff
@@ -4500,6 +4516,13 @@ msgstr ""
 msgid "Unknown error"
 msgstr ""
 
+#. module: mass_mailing
+#. odoo-python
+#: code:addons/mass_mailing/controllers/main.py:0
+#, python-format
+msgid "Unsubscribe"
+msgstr ""
+
```

Comment:
Hello, thanks for trying to keep the pot file up to date, but in the future please be more cautious about not duplicating a term (there was already an "Unsubscribe" in this pot file). The easiest way to do this is to re-export the pot file and copy+paste it into the existing one and then check the diff for which changes seem to make sense. Duplicated terms break the Transifex sync :crying_cat_face:

I'll fix it this time around